lib/galaxy/webapps/galaxy/api/jobs.py

```diff
@@ -185,6 +186,16 @@ def outputs( self, trans, id, **kwd ):
         job = self.__get_job( trans, id )
         return self.__dictify_associations( trans, job.output_datasets, job.output_library_datasets )
 
+    @expose_api_anonymous
+    def build_for_rerun( self, trans, id, **kwd ):
```

Comment:
Good call, I knew I meant to go back and do something else after talking to @guerler about this :)